controllers/disruption_controller.go

```diff
@@ -210,9 +217,7 @@ func (r *DisruptionReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		// the injection is being created or modified, apply needed actions
 		controllerutil.AddFinalizer(instance, chaostypes.DisruptionFinalizer)
 		if err := r.Update(context.Background(), instance); err != nil {
-			r.log.Errorw("error adding disruption finalizer", "error", err)
-
-			return ctrl.Result{Requeue: true}, err
+			return ctrl.Result{}, fmt.Errorf("error adding disruption finalizer: %w", err)
```

Comment:
why is this no longer a `Requeue: true`?

Reply:
FYI https://github.com/DataDog/chaos-controller/pull/687#discussion_r1162354861